src/idl_gen_go.cpp

```diff
@@ -103,10 +103,12 @@ class GoGenerator : public BaseGenerator {
     bool needs_imports = false;
     for (auto it = parser_.enums_.vec.begin(); it != parser_.enums_.vec.end();
```

Comment:
It would be perfect if you could make this for loops range based for more readability.

Reply:
That can be done in a clean up PR, don't need to delay this fix for that. But yes, we should modernize all of our loops.